packages/ddex/ingester/parser/parser.go

```diff
@@ -354,6 +354,13 @@ func (p *Parser) parseBatch(batch *common.UnprocessedBatch, deliveryRemotePath s
 
 		// Validate the URL without the prefix "/"
 		releaseURL := strings.TrimPrefix(safeInnerText(messageInBatch.SelectElement("URL")), "/")
+
+		// Special case for Fuga deliveries with a different URL format
+		if strings.Contains(releaseURL, "ddex-prod-fuga-raw") {
```

Comment:
If this continues to be our pattern, it would be good to break this out into a separate function that is basically a giant switch statement. I imagine we need to establish some kind of pattern for these things